SIL.Core.Tests/Xml/XmlUtilsTests.cs

```diff
@@ -176,6 +176,36 @@ public void WriteNode_DoesNotIndentChildWhenSuppressed()
 			}
 			Assert.That(output.ToString(), Is.EqualTo(expectedOutput));
 		}
+
+		[Test]
+		public void WriteNode_PreserveNamespacesArePreserved()
+		{
+			string input = @"<text><span class='bold' xml:space='preserve'> </span></text>";
+			string expectedOutput =
+				"<?xml version=\"1.0\" encoding=\"utf-16\"?>\r\n"
+				+ "<root>\r\n"
+				+ "	<text>\r\n"
+				+ "		<span\r\n"
+				+ "			class=\"bold\"\r\n"
+				+ "			 xml:space=\"preserve\"> </span>\r\n"
```

Comment:
I'm assuming this is what you want to keep around, but I'm not clear what it would look like otherwise, would it just be `space=`?

Reply:
yes, without the xml namespace the standard writer will ignore the 'space="preserve"' and drop that content.